src/Prowlarr.Api.V1/ProviderResource.cs

```diff
@@ -39,8 +39,8 @@ public virtual TProviderResource ToResource(TProviderDefinition definition)
                 Fields = SchemaBuilder.ToSchema(definition.Settings),
 
                 //prowlarr/supported#{0} are custom build redirect pages; if passing a new var, create a new redirect
-                InfoLink = string.Format("https://wikijs.servarr.com/prowlarr/supported#{0}",
-                    definition.Implementation.ToLower())
+                InfoLink = string.Format("https://wikijs.servarr.com/prowlarr/supported-indexers#{0}",
```

Comment:
Need to Revert this change as it applies to all.

need to conditionalize this to be for indexers only